.travis.yml

```diff
@@ -1,27 +1,30 @@
 language: php
 
-php:
-# - 5.3 # requires old distro, see below
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
-  - hhvm # ignore errors, see below
-
-# lock distro so new future defaults will not break the build
-dist: trusty
-
 matrix:
   include:
-    - php: 5.3
-      dist: precise
-  allow_failures:
-    - php: hhvm
+    - dist: precise
+      php: 5.3
+    - dist: trusty
+      php: 5.4
+    - dist: trusty
+      php: 5.5
+    - dist: xenial
+      php: 5.6
+    - dist: xenial
+      php: 7.0
+    - dist: bionic
+      php: 7.1
+    - dist: bionic
+      php: 7.2
+    - dist: bionic
+      php: 7.3
+    - dist: bionic
+      php: 7.4
+    - dist: trusty
+      php: hhvm-3.18
 
 install:
-  - composer install --no-interaction
+  - travis_retry composer install
```

Comment:
👆

Reply:
I'd strongly recommend to leave this in. The only reason you haven't seen this yet is because there are so few builds on this repo, but most other projects have `travis_retry` enabled, and I've seen it actually be needed.